package.json

```diff
@@ -16,58 +16,52 @@
     "npm-pack": "cd dist/ngx-location-picker && npm pack",
     "build-lib": "ng build ngx-location-picker && npm run copy-files && npm run npm-pack"
   },
-  "peerDependencies": {
-    "@acpaas-ui-widgets/ngx-location-viewer": "^1.2.0",
-    "@acpaas-ui/ngx-flyout": "^4.6.1",
-    "@acpaas-ui/ngx-leaflet": "^4.6.1",
-    "@angular/common": "^6.0.0",
-    "@angular/core": "^6.0.0",
-    "@angular/forms": "^6.0.0"
+  "peerDependencies": {},
+  "dependencies": {
+    "@acpaas-ui-widgets/ngx-location-viewer": "^1.0.0",
+    "@acpaas-ui/ngx-icon": "^5.0.0",
+    "@acpaas-ui/ngx-leaflet": "^5.0.2",
+    "@angular/common": "^8.2.14",
```

Comment:
de angular packages dienen in peerdependencies te blijven denk ik (en idealiter hebben ze een bredere compatibiliteit qua versie)

Reply:
Er was zelfs meer aan de hand, maar ik kan pas pushen als de location viewer gereleased is, aangezien die wél een dependency is.

Reply:
Fixed.